### PR TITLE
done the schedule

### LINKED ISSUE #2: Update the UG
image

### PR DESCRIPTION
**Pull request by KevinLohJunYong**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+Adds a new schedule into the schedule list.
+
+Format: `add schedule`
+
+Prompts: `start date and time`, `end date and time`, `[tags]`
+
+Example(s):
+* `add schedule meeting 2021-02-15,2100 2021-02-15,2300`
+* `add schedule consultation 2021-02-15,2300 2021-02-15,2400 consultation
+  `
```
Comment (nicholastanvis) [inline docs/UserGuide.md]: this backtick should be at the end of line

File: docs/UserGuide.md [hunk 2]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+Deletes an existing schedule with the specified name in the schedule list.
+
+Format: `delete a schedule`
+
+Prompt: `name`
+
+Example(s):
+* `delete schedule meeting`
+
+
```
Comment (nicholastanvis) [inline docs/UserGuide.md:111]: missing triple dash

File: docs/UserGuide.md [hunk 3]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+Format: `find schedule`
+* `The search is case-insensitive e.g. meeting will match Meeting`
+
+Prompt: `name`
+
+Example(s):
+* `finding schedule meeting`
+
+### Listing schedules today (of the same week)
+List schedules today/this week
```
Comment (nicholastanvis) [inline docs/UserGuide.md]: missing full stop

File: docs/UserGuide.md [hunk 4]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+
+Format: `list schedule`
+
+Prompt: `by day/week`
```
Comment (nicholastanvis) [inline docs/UserGuide.md:95]: change this to `by day`, `by week`

Comment (nicholastanvis) [inline docs/UserGuide.md:95]: and Prompts

File: docs/UserGuide.md [hunk 5]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+
+Example(s):
+* `list schedule by day`
+* `list schedule by week`
+
+### Deleting a schedule
+Deletes an existing schedule with the specified name in the schedule list.
+
+Format: `delete a schedule`
```
Comment (nicholastanvis) [inline docs/UserGuide.md]: format should be `delete schedule`

**Review by nicholastanvis — CHANGES_REQUESTED** [review]

**Review by tsh22 — APPROVED** [review]

File: docs/UserGuide.md [hunk 6]
```diff
@@ -65,6 +65,50 @@ Example(s):
...
+### Finding a schedule
+Finds an existing schedule by name in the schedule list.
+
+Format: `find schedule`
+* `The search is case-insensitive e.g. meeting will match Meeting`
+
+Prompt: `name`
+
+Example(s):
+* `finding schedule meeting`
```
Comment (nicholastanvis) [inline docs/UserGuide.md]: find

File: docs/UserGuide.md [hunk 7]
```diff
@@ -92,7 +136,7 @@ Lists all the tasks by the specified module/today/this week.
 
 Format: `list tasks`
 
-Prompt: `by module`, `by day`, `by week`
+Prompts: `by module`, `by day`, `by week`
```
Comment (nicholastanvis) [inline docs/UserGuide.md:139]: change back to Prompt: `by module/day/week`

**Review by nicholastanvis — CHANGES_REQUESTED** [review]

**Review by nicholastanvis — APPROVED** [review]

**Review by goatygoatygoat — APPROVED** [review]

**Pull request merged**